<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QClientClass</class>
 <widget class="QMainWindow" name="QClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>452</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>QClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>253</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScrollMargin">
     <number>18</number>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="indentation">
     <number>10</number>
    </property>
    <property name="uniformRowHeights">
     <bool>true</bool>
    </property>
    <property name="itemsExpandable">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="headerHidden">
     <bool>false</bool>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>30</y>
      <width>253</width>
      <height>192</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_Text"/>
    <addaction name="actionCut_Text"/>
    <addaction name="actionPaste_Text"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <widget class="QMenu" name="menuText">
    <property name="title">
     <string>Text</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionStrikeout"/>
    <addaction name="separator"/>
    <addaction name="actionColor"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menuNodes">
    <property name="title">
     <string>Nodes</string>
    </property>
    <addaction name="actionAdd_New_Node"/>
    <addaction name="actionAdd_New_Subnode"/>
    <addaction name="actionDelete_Node"/>
    <addaction name="actionOpen_Note"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNodes"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuText"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_New_Node"/>
   <addaction name="actionAdd_New_Subnode"/>
   <addaction name="actionOpen_Note"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionCopy_Text">
   <property name="text">
    <string>Copy Text</string>
   </property>
  </action>
  <action name="actionCut_Text">
   <property name="text">
    <string>Cut Text</string>
   </property>
  </action>
  <action name="actionPaste_Text">
   <property name="text">
    <string>Paste Text</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionStrikeout">
   <property name="text">
    <string>Strikethrough</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionAdd_New_Node">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/alexa/OneDrive/Pictures/Desktop/a se sterge regulat/n.png</normaloff>C:/Users/alexa/OneDrive/Pictures/Desktop/a se sterge regulat/n.png</iconset>
   </property>
   <property name="text">
    <string>Add New Node</string>
   </property>
  </action>
  <action name="actionAdd_New_Subnode">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/alexa/OneDrive/Pictures/Desktop/a se sterge regulat/c.png</normaloff>C:/Users/alexa/OneDrive/Pictures/Desktop/a se sterge regulat/c.png</iconset>
   </property>
   <property name="text">
    <string>Add New Subnode</string>
   </property>
  </action>
  <action name="actionDelete_Node">
   <property name="text">
    <string>Delete Node</string>
   </property>
  </action>
  <action name="actionOpen_Note">
   <property name="icon">
    <iconset>
     <normaloff>../photos/open.png</normaloff>../photos/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Note</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QClient.qrc"/>
 </resources>
 <connections/>
</ui>
